src/util/snapshots.tsx

```diff
@@ -0,0 +1,78 @@
+import { LxdInstance } from "types/instance";
+import { TestFunction } from "yup";
+import { AnyObject } from "yup/lib/types";
+import { checkDuplicateName, getTomorrow } from "./helpers";
+
+export interface SnapshotFormValues {
+  name: string;
+  stateful: boolean;
+  expirationDate: string | null;
+  expirationTime: string | null;
+}
+
+export const getExpiresAt = (
+  expirationDate: string,
+  expirationTime: string | null
```

Comment:
All callers of this function make the expiration time mandatory. We should either remove the null here, or allow the callers to pass in a data string and a null time.

Reply:
Right, I forgot to update this after making the `expirationTime` mandatory.